apps/src/code-studio/components/AdvancedShareOptions.jsx

```diff
@@ -137,21 +146,29 @@ class AdvancedShareOptions extends React.Component {
         <textarea
           type="text"
           onClick={e => e.target.select()}
-          readOnly="true"
+          readOnly={true}
           value={iframeHtml}
           style={style.embedInput}
+          aria-label={i18n.codeToEmbed()}
         />
-        <label style={{display: 'flex'}}>
+        <div>
           <input
+            id="embedCodeCheckbox"
             type="checkbox"
-            style={{accentColor: color.brand_primary_default}}
+            style={style.embedCodeCheckbox}
             checked={this.state.embedWithoutCode}
+            aria-label={i18n.hideAbilityToViewCode()}
```

Comment:
Do you need this when you already have a label for the input?

Reply:
nope! good catch. thanks